docs/AutomatedTests.md

```diff
@@ -15,19 +15,38 @@ test libraries that contain mock implementations of system APIs such as kernel s
 allowing unit tests to run entirely in user mode. Unit tests are run with address sanitization on,
 to catch memory issues.
 
+Tests in this category currently include:
+* bpf2c_tests.exe: This unit tests bpf2c conversion.
+* netebpfext_unit.exe: This unit tests netebpfext logic.
+* unit_tests.exe: This unit tests logic in ebpfcore, libbpf, and netsh.
+
 ## Kernel tests
 Whereas unit tests run in user-mode and test the static libraries, kernel tests exercise the actual
 runtime binaries running on a real machine. CI/CD uses [self-hosted runners](SelfHostedRunnerSetup.md)
 for this purpose. Such tests can catch bugs in lightweight scaffolding code, installation issues,
 etc. and potentially help identify any gaps in the mock implementations of system APIs if any bugs
 are found that should have been caught by unit tests.
 
+Tests in this category currently include:
+* api_test.exe: This tests user mode APIs to interact with eBPF.
+* connect_redirect_tests.exe: This tests connection redirection functionality.
+* sample_ext_app.exe: This a sample extension driver.
```

Comment:
nit: `This tests the sample extension driver.`

Reply:
Done